### PR TITLE
[4.0] Remove Batch tooltips

### PR DESCRIPTION
**Pull request by brianteeman**
We're not using bootstrap tooltips and when reviewing the tooltips for the fields in the batch modals it is clear to see that they are superfluous.

This PR removes them and the associated language strings

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b424002c09812cf9104e7f48670278d0d0aad8aa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25605">issues.joomla.org/tracker/joomla-cms/25605</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on b424002c09812cf9104e7f48670278d0d0aad8aa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25605">issues.joomla.org/tracker/joomla-cms/25605</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

File: administrator/components/com_banners/Service/Html/Banner.php [hunk 1]
```diff
@@ -35,9 +35,7 @@ public function clients()
 		return implode(
 			"\n",
 			array(
-				'<label id="batch-client-lbl" for="batch-client" class="hasTooltip" title="'
-					. HTMLHelper::_('tooltipText', 'COM_BANNERS_BATCH_CLIENT_LABEL', 'COM_BANNERS_BATCH_CLIENT_LABEL_DESC')
```
Comment (wilsonge) [inline administrator/components/com_banners/Service/Html/Banner.php]: I think this is the last use of COM_BANNERS_BATCH_CLIENT_LABEL_DESC

File: language/en-GB/en-GB.lib_joomla.ini [hunk 2]
```diff
@@ -352,22 +352,18 @@ JLIB_HTML_ACCESS_SUMMARY="Summary."
 JLIB_HTML_ADD_TO_ROOT="Add to root."
 JLIB_HTML_ADD_TO_THIS_MENU="Add to this menu."
 JLIB_HTML_BATCH_ACCESS_LABEL="Set Access Level"
-JLIB_HTML_BATCH_ACCESS_LABEL_DESC="Not making a selection will keep the original access levels when processing."
```
Comment (wilsonge) [inline language/en-GB/en-GB.lib_joomla.ini:355]: Is this intuitive in the batch screens? I get it's probably not best placed in a tooltip. But we might want to make it clear in another way

Comment (brianteeman) [reply language/en-GB/en-GB.lib_joomla.ini:355]: Sorry I thought I replied to this.
Yes its perfectly clear to me as it is after this PR. See
image

Adding the tooltip which says ""Not making a selection will keep the original access levels when processing." is just duplicating the info

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**